_drafts/bridging-digital-and-physical.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout:       post
+title:        Bridging digital and physical with a MakerBot Replicator 2
+author:       Dave Mills
+summary:
+image:        http://image.shutterstock.com/display_pic_with_logo/1451378/316560056/stock-vector-pumpkin-dressed-up-in-stripy-suit-furry-art-character-halloween-greeting-design-316560056.jpg
```

Comment:
Download it from Shutterstock (credentials are in Meldium but might be outdated. @i8ramin can get it for you if that's the case), then log into Cloudinary (Meldium again) and upload to the engineering folder. It'll give you a public URL. 

Alternately, come find me and I'll show you.